src/routes/connect.svelte

```diff
@@ -5,7 +5,8 @@
   import GetInTouchForm from "@/components/sections/get-in-touch-form.svelte";
   import CallToActionSection from "@/components/sections/call-to-action-section.svelte";
 
-  const googleFormUrl = `https://docs.google.com/forms/d/e/1FAIpQLSfJanOAaL2mdjpf193tFeCClBzpW_COEO_crAE8hqsJCB_Rwg/viewform?embedded=true`;
+  const googleFormId =
+    "1FAIpQLSfJanOAaL2mdjpf193tFeCClBzpW_COEO_crAE8hqsJCB_Rwg";

```

Comment:
The form ID itself can also be stored in a constants file elsewhere in the project and imported by the `get-in-touch-form.svelte`.